index.bs

```diff
@@ -279,5 +512,145 @@ Threat model {#security-threat}
 Mitigations and defenses {#security-defenses}
 ---------------------------------------------
 
+Appending A: Messages
+=====================
+
+<pre>
+; type key 14
+presentation-url-availability-request = {
+  request
+  1: [* text] ; urls
+  2: microseconds ; watch-duration
+  3: int ; watch-id
+}
+
+; type key 15
+presentation-url-availability-response = {
+  response
+  1: [* url-availability] ; url-availabilities
+}
+
+; type key 103
```

Comment:
Why the jump in key numbers?

Reply:
I added a comment explaining (Lower type keys are used for messages we expect to be more frequent and higher type keys are used for less frequently used messages)